lib/messages.js

```diff
@@ -1,10 +1,20 @@
 module.exports = {
-  userConfig: (path) => `Couldn't read the userConfig from ${path}!`,
   breakpoints: (path) =>
-    `Couldn't read drupal's breakpoints file from ${path}!`,
-  writeCSSFile: (path) => `Couldn't write css file to ${path}!`,
-  writeJSFile: (path) => `Couldn't write js file to ${path}!`,
-  readFile: (path) => `Couldn't read from ${path} file!`,
+    `Couldn't read Drupal's breakpoints file from ${path}!`,
+  defaultThemeName: (name) =>
+    `Missing "userConfig.drupal.themeName". Will use "${name}" for now.`,
```

Comment:
This message is also no longer accurate since the default theme name is now an empty string.